palamar/settings.py

```diff
@@ -195,6 +195,9 @@
 # file location
 FILE_STORAGE = os.path.join(BASE_DIR, "files")
 
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
There is "BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))" config variable on top of the settings.py. Do we need this line?

Also there is no need for this line because STATIC_ROOT does not require it.